GiEnJul/ClientApp/src/components/admin/overview/Statistics.tsx

```diff
@@ -0,0 +1,88 @@
+import { Card, Grid, Typography } from "@material-ui/core";
+import React from "react";
+import { GiverType, RecipientType, SelectedConnectionType } from "./Types";
+import { FiberManualRecord } from "@material-ui/icons";
+import useStyles from "./Styles";
+
+type IStatistics = {
+  givers: GiverType[] | [];
+  recipients: RecipientType[] | [];
+};
+
+const Statistics: React.FC<IStatistics> = ({ givers, recipients }) => {
+  const totalGivers: number = givers.length;
+  const totalRecipints: number = recipients.length;
+  let suggestedMatch: number = 0;
```

Comment:
Agree with Joachim, use filter to set these values
https://stackoverflow.com/a/47377879